src/viur/core/utils/json.py

```diff
@@ -47,7 +47,14 @@ def preprocess(obj: t.Any) -> t.Any:
             return tuple(ViURJsonEncoder.preprocess(value) for value in obj)
 
         elif hasattr(obj, "__class__") and obj.__class__.__name__ == "SkeletonInstance":  # SkeletonInstance
-            return {bone_name: ViURJsonEncoder.preprocess(obj[bone_name]) for bone_name in obj}
+            data = obj.dbEntity
+            if data is None:
+                data ={}
+            return {
+                ".__entity__": ViURJsonEncoder.preprocess(dict(data)),
```

Comment:
`db.Entity` inherits from `dict`. I think this is obsolete, but please test.
```suggestion
                ".__entity__": ViURJsonEncoder.preprocess(data),
```